src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,90 @@
+package codeu.chat.util;
+
+import java.lang.StringBuilder;
+import java.io.IOException;
+
+// This utility class allows us to read input that may contain
+// multiple tokens to allow for multi-token commands in our
+// application later on.
+public final class Tokenizer {
+
+  // The following instance variables help keep track of the current token
+  // we are building (token), the original input (source), and the current
+  // index we are at in our source (at).
+  private StringBuilder token;
+  private String source;
+  private int at;
```

Comment:
I thought this variable name was a little vague as well, but I just decided to go with it since it was from the tutorial. I now know next time to just make things descriptive since that's a better decision in the long run.